projects/irssi/build.sh

```diff
@@ -18,8 +18,10 @@
 # configure script needs leak checking disabled to not fail
 export ASAN_OPTIONS=detect_leaks=0
 ./autogen.sh
+# TODO: Stop using LIB_FUZZING_ENGINE_DEPRECATED and make this build use
+# LIB_FUZZING_ENGINE.
```

Comment:
extend the comment with something like ", which is a compilation flag, not a path to a library", to give more context in case anyone else would notice that and attempt to fix

Reply:
and for the other files as well

Reply:
I added a link to an issue giving full context.